src/aui/auibar.cpp

```diff
@@ -2248,6 +2248,10 @@ void wxAuiToolBar::DoIdleUpdate()
     }
 }
 
+wxSize wxAuiToolBar::DoGetBestSize() const
+{
+    return GetClientSize();
```

Comment:
It's not a good idea to return the current size from best size because it means that if the window grows, it can never be shrunk back. Couldn't we use `RealizeHelper()` here to get the real best size?